datacapture/src/main/java/com/google/android/fhir/datacapture/mapping/ResourceMapper.kt

```diff
@@ -161,9 +170,17 @@ object ResourceMapper {
       structureMapProvider?.let { it(questionnaire.targetStructureMap!!, simpleWorkerContext) }
         ?: return Bundle()
 
+    val structureMapUtilities =
+      if (transformSupportServices == null) StructureMapUtilities(simpleWorkerContext)
+      else StructureMapUtilities(simpleWorkerContext, transformSupportServices)
```

Comment:
```suggestion
  StructureMapUtilities(simpleWorkerContext, transformSupportServices)
  .transform(simpleWorkerContext, questionnaireResponse, structureMap, this)
```
Since passing null transformSupportServices is fine in the constructor.

Reply:
Agree! I have updated this